extensions/shared/is-wpcom-site.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * External Dependencies
+ */
+import { get } from 'lodash';
+
+export default function isWpcomSite() {
+	return (
+		get( 'object' === typeof window ? window : null, [ '_currentSiteType' ], null ) === 'simple'
```

Comment:
This may be a little hard to read. Think I'd pull the ternary out of the `get` call e.g.

```
if ( 'object' === typeof window ) {
    return get( window, [ '_currentSiteType' ] ) === 'simple';
}

return false;
```

or even

```
'object' === typeof window && get( window, [ '_currentSiteType' ] ) === 'simple'
```

Reply:
and like Jon mentioned down below, don't need `get` at all here :-)

Reply:
Better now, thanks! :-)